vlib/v/fmt/testdata/vmodules/src_import/src/a/aa/import_without_src_expected.vv

```diff
@@ -0,0 +1,4 @@
+// NOTE: A test that follows a directory strucutre and module naming like this one
+// also tests whether vfmt would add a prefix to the import.
+
+import a.ab
```

Comment:
Why is the `src.` removed at all?

The less vfmt modifies the paths, the better.

Reply:
So you say to allow for both without a formatting error?
For usage of the module both paths work.

Reply:
I'd prefer the "one way" philo here when thinking about it.

Reply:
My main concern about it, is preventing oscillations, where it is removed by one run of vfmt, based on a heuristic, and then added in another run of vfmt, based on another heuristic. It seems to me safer to not modify the imports at all/not have special cases for it.

Reply:
> So you say to allow for both without a formatting error?

Imho vfmt should not try to outsmart the person writing the code. It should just format the code as it is written.

Reply:
Removing the special case for the vmodules is very important though, and `src.` is an edge case, that will not be encountered very often, so overall, excellent work.

Reply:
> Personally, I would have preferred ALWAYS stripping `src.`. It is supposed to be a "silent" subdir, same as "modules".

There are edge cases where `src.` is part of the import and required to be kept for the module to work. E.g. importing from a module that is structured like this issue is mentioning https://github.com/vlang/v/issues/19799. We might improve module lookup so that it works without it then we can always strip `src.`.